Google-Maps-iOS-Utils.podspec

```diff
@@ -14,17 +14,15 @@ Pod::Spec.new do |s|
   s.homepage     = "https://github.com/googlemaps/google-maps-ios-utils"
   s.license      = { :type => 'Apache 2.0', :file => 'LICENSE' }
   s.authors      = "Google Inc."
-  s.platform     = :ios, '9.0'
+  s.platform     = :ios, '13.0'
   s.source       = { :git => "https://github.com/googlemaps/google-maps-ios-utils.git",
                      :tag => "v#{s.version.to_s}" }
   s.requires_arc = true
   s.module_name = "GoogleMapsUtils"
   s.swift_version = '5.0'
 
-  s.dependency 'GoogleMaps'
+  s.dependency 'GoogleMaps', '>= 7.3'
```

Comment:
Just requiring iOS 13 will make `pod lint` succeed (as it will use the latest GoogleMaps matching the minimum iOS version, meaning currently 7.3), but with `EXCLUDED_ARCHS` removed build successfully will require GoogleMaps 7.3 anyway so it is better to make it explicit.